packages/transaction-controller/src/TransactionController.ts

```diff
@@ -940,7 +930,7 @@ export class TransactionController extends BaseController<
 
   readonly #trace: TraceCallback;
 
-  readonly #transactionHistoryLimit: number;
+  // readonly #transactionHistoryLimit: number;
```

Comment:
This isn't related to the history properties, but how many transactions we retain. We need to keep this.